Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.5.1-slim
+FROM ruby:2.6.2-slim
 
 RUN apt-get update && \
     apt-get install -y build-essential nodejs libpq-dev postgresql-client imagemagick
```

Comment:
@frankyston I'll kindly ask you to update ruby and rails in two separate PRs. 

I just notice the package `postresql-client` have changed on `debian:stretch-slim` version used by `ruby:2.6.2-slim` image. without the proper fix, current call4paperz docker image does not build.

a subset of the stack trace when trying to build the image:

```
Errors were encountered while processing:
 postgresql-client-9.6
 postgresql-client
E: Sub-process /usr/bin/dpkg returned an error code (1)
ERROR: Service 'web' failed to build: The command '/bin/sh -c apt-get update &&     apt-get install -y build-essential nodejs libpq-dev postgresql-client imagemagick' returned a non-zero code: 100
```

Reply:
@lucasrenan Ok, I'm going to do this in just the rails and ruby in another PR.